### DISCUSSION TITLE
When support openssl 1.1.1

### DISCUSSION THREAD

**Discussion by oyljerry** [Azure/azure-iot-sdk-c#2077] [Ideas]
Hi All:  
    I found current SDK is used with openssl 1.0.2s, Do you have any plan to upgrade code to support openssl 1.1.1 version? Thanks.

**Comment by danewalton-msft** [comment]
This is not a priority at this time and I'd like to move it to discussions as a feature request.

**Comment by ericwolz** [comment]
Openssl 3 support was added with release [LTS_07_2022_Ref02]
